gilbreth_support/urdf/gilbreth.xacro

```diff
@@ -1,116 +1,63 @@
 <?xml version="1.0" ?>
 <robot name="gilbreth" xmlns:xacro="http://www.ros.org/wiki/xacro">
 
-  <!-- linear arm actuator -->
-  <xacro:include filename="$(find gilbreth_support)/urdf/linear_actuator/linear_actuator.xacro" />
-  <xacro:linear_actuator prefix="linear_actuator_"/>
-
-  <!-- include the robot xacro -->
-  <xacro:include filename="$(find ur_description)/urdf/ur10.urdf.xacro" />
-  <xacro:ur10_robot prefix="robot_" joint_limited="false"/>
-
-  <!-- vacuum gripper -->
-  <xacro:property name="vacuum_gripper" value="vacuum_gripper_" />
-  <xacro:include filename="$(find gilbreth_support)/urdf/vacuum_gripper/vacuum_gripper.xacro"/>
-  <xacro:vacuum_gripper prefix="vacuum_gripper_"/>
+  <!-- robot system -->
+  <xacro:include filename="$(find gilbreth_support)/urdf/robot_system/robot_system.xacro"/>
 
   <!-- bins -->
-  <xacro:include filename="$(find gilbreth_support)/urdf/bin/bin.xacro"/>
+<!--  <xacro:include filename="$(find gilbreth_support)/urdf/bin/bin.xacro"/>
   <xacro:bin prefix="bin_1_"/>
   <xacro:bin prefix="bin_2_"/>
-  <xacro:bin prefix="bin_3_"/>
+  <xacro:bin prefix="bin_3_"/>-->
```

Comment:
Is this commented block still needed?

Reply:
I intend on implementing the `osrf_gear/ObjectDisposalPlugin` with the bins, so that picked parts are removed from gazebo when they are dropped in the bins. I didn't get around to fully debugging that for this PR, so I just left it commented for now